src/VirtualClient/VirtualClient.Main/VirtualClient.Main.csproj

```diff
@@ -176,6 +180,7 @@
         <LinuxProfiles Include="profiles\PERF-CPU-LAPACK.json" />
         <LinuxProfiles Include="profiles\PERF-STRESSNG.json" />
         <LinuxProfiles Include="profiles\PERF-MEM-LMBENCH.json" />
+		<LinuxProfiles Include="profiles\PERF-MYSQL-SYSBENCH-OLTP.json" />
```

Comment:
nit: spacing

Reply:
hmm why the spacing still looks off?

Reply:
not sure...it looks fine in vs but here the spacing is weird throughout the file. if i try to fix the spacing in vs it won't let me commit (it doesn't register that anything changes). should i try to fix it in the github editor?